### ISSUE TITLE
Integration broken with 2023.02.00

### ISSUE THREAD

**Issue by mxwi** [binsentsu/home-assistant-solaredge-modbus#136]
Hi,

the integration ist broken with 2023.02.00

```
2023-02-02 07:14:14.726 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration solaredge_modbus which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-02-02 07:16:08.571 ERROR (MainThread) [homeassistant.loader] Unexpected exception importing component custom_components.solaredge_modbus
File "/config/custom_components/solaredge_modbus/__init__.py", line 10, in <module>
2023-02-02 07:16:08.592 ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration solaredge_modbus: Unable to import component: Exception importing custom_components.solaredge_modbus
2023-02-02 07:24:45.093 ERROR (MainThread) [homeassistant.loader] Unexpected exception importing platform custom_components.solaredge_modbus.application_credentials
File "/config/custom_components/solaredge_modbus/__init__.py", line 10, in <module>
2023-02-02 07:24:48.188 ERROR (MainThread) [homeassistant.loader] Unexpected exception importing component custom_components.solaredge_modbus
File "/config/custom_components/solaredge_modbus/__init__.py", line 10, in <module>
File "/config/custom_components/solaredge_modbus/__init__.py", line 10, in <module>
ImportError: Exception importing custom_components.solaredge_modbus
2023-02-02 07:24:56.424 ERROR (MainThread) [homeassistant.loader] Unexpected exception importing platform custom_components.solaredge_modbus.application_credentials
File "/config/custom_components/solaredge_modbus/__init__.py", line 10, in <module>
2023-02-02 07:24:57.940 ERROR (MainThread) [homeassistant.loader] Unexpected exception importing component custom_components.solaredge_modbus
File "/config/custom_components/solaredge_modbus/__init__.py", line 10, in <module>
File "/config/custom_components/solaredge_modbus/__init__.py", line 10, in <module>
ImportError: Exception importing custom_components.solaredge_modbus
2023-02-02 07:25:06.165 ERROR (MainThread) [homeassistant.loader] Unexpected exception importing platform custom_components.solaredge_modbus.config_flow
File "/config/custom_components/solaredge_modbus/__init__.py", line 10, in <module>
2023-02-02 07:25:06.178 ERROR (MainThread) [homeassistant.config_entries] Error occurred loading configuration flow for integration solaredge_modbus: Exception importing custom_components.solaredge_modbus.config_flow
```
```

2023-02-02 07:16:08.571 ERROR (MainThread) [homeassistant.loader] Unexpected exception importing component custom_components.solaredge_modbus
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 730, in get_component
    cache[self.domain] = importlib.import_module(self.pkg_path)
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/config/custom_components/solaredge_modbus/__init__.py", line 10, in <module>
    from pymodbus.client.sync import ModbusTcpClient
  File "/usr/local/lib/python3.10/site-packages/pymodbus/client/sync.py", line 166, in <module>
    class ModbusTcpClient(BaseModbusClient):
  File "/usr/local/lib/python3.10/site-packages/pymodbus/client/sync.py", line 170, in ModbusTcpClient
    def __init__(self, host='127.0.0.1', port=Defaults.Port,
AttributeError: type object 'Defaults' has no attribute 'Port'
```

Home Assistant 2023.2.0
Supervisor 2023.01.1
Operating System 9.3
Frontend 20230201.0 - latest

**Comment by mxwi** [comment]
Guess it is more an issue with the modbus integration in HA in general.

**Comment by binsentsu** [comment]
Hi. The modbus library bump to new major version was the reason. Issue should be resolved in release 1.8.0
